dev-libs/libunistring/libunistring-1.3.recipe

```diff
@@ -50,14 +50,9 @@ defineDebugInfoPackage libunistring$secondaryArchSuffix \
 
 BUILD()
 {
-	autoreconf -fi
-	if [ -n "$secondaryArchSuffix" ]; then
-		runConfigure ./configure --disable-static
-	else
-		runConfigure --omit-dirs docDir ./configure \
-			--disable-static \
-			--docdir="$developDocDir"
-	fi
+	runConfigure --omit-dirs docDir ./configure \
+		--disable-static \
+		--docdir=$developDir/documentation/libunistring
```

Comment:
When the documentation isn't separated into an any-arch subpackage, this special handling of `$docDir` and `$developDocDir` isn't strictly necessary, but it doesn't harm either, so I think it's fine.